### PR TITLE
docs: update how to generate sample config

### PR DESCRIPTION
**Pull request by sspaink**
Match what we have on the docs side: https://docs.influxdata.com/telegraf/v1.21/introduction/getting-started/#create-and-edit-the-configuration-file

Not exactly the same, I made this doc less wordy because it is just configuration and not a "getting started" like the website. I have no issue having them match exactly either though.

### CODE REVIEW

File: docs/CONFIGURATION.md [hunk 1]
```diff
@@ -12,14 +12,20 @@ View the default [telegraf.conf][] config file with all available plugins.
...
-telegraf config > telegraf.conf
+telegraf --sample-config > telegraf.conf
 ```
 
 To generate a file with specific inputs and outputs, you can use the
 --input-filter and --output-filter flags:
 
 ```sh
-telegraf --input-filter cpu:mem:net:swap --output-filter influxdb:kafka config
+telegraf --sample-config --input-filter cpu:mem:net:swap --output-filter influxdb:kafka config
```
Comment (Hipska) [inline docs/CONFIGURATION.md:22]:
```suggestion
telegraf --sample-config --input-filter cpu:mem:net:swap --output-filter influxdb:kafka
```

**Review by Hipska — CHANGES_REQUESTED** [review]

File: docs/CONFIGURATION.md (hunk 1 above)
Comment (Hipska) [reply docs/CONFIGURATION.md:22]: Yep 👍

**Review by Hipska — APPROVED** [review]
Note that the commands are also explained in other places:

https://github.com/influxdata/telegraf/blob/828b0400ac16a83ade37ed8632d330ca09e6115d/docs/COMMANDS_AND_FLAGS.md?plain=1#L46-L53

These need to be updated as well..

**Review by MyaLongmire — APPROVED** [review]

**Comment by sspaink** [comment]
After some discussion closing this because this should be an issue so that the following works:
`telegraf config --input-filter cpu --output-filter influxdb`

instead of `telegraf --input-filter cpu --output-filter influxdb config`

`--sample-config` should probably be deprecated as well in favor of `config`

**Comment by Hipska** [comment]
According to the changes you made here, I assumed the `config` command would become deprecated.

**Comment by sspaink** [comment]
I originally was in favor of doing it that way so sorry for the confusion, but looking at it again using `config` does seem like a more standard CLI way of doing things using a command followed by flags instead of a flag followed by more flags.

**Comment by Hipska** [comment]
That makes more sense indeed!

**Pull request closed without merging**